view/frontend/web/js/view/payment/method-renderer/omise-cc-method.js

```diff
@@ -257,6 +257,24 @@ define(
                             redirectOnSuccessAction.execute();
                         }
                     });
+            },
+
+            getSelectedTokenBillingAddress: function() {
+                var selectedBillingAddress = quote.billingAddress();
+                var address = {
+                    state          : selectedBillingAddress.region,
```

Comment:
Do we have existing formatting rules for this?

Reply:
Thanks

Reply:
Done. Reformatting this as it should be more readable.